reana_server/cli.py

```diff
@@ -97,3 +104,92 @@ def users_create_default(email, organization_name, id_):
     except Exception as e:
         click.echo('Something went wrong: {0}'.format(e))
         sys.exit(1)
+
+
+@users.command(
+    'get',
+    help='Get information of users matching search criteria.')
+@click.option(
+    '--id',
+    help='The id of the user.')
+@click.option(
+    '-e',
+    '--email',
+    help='The email of the user.')
+@click.option(
+    '-ut',
+    '--user-token',
+    help='The REANA_ACCESS_TOKEN of the user.')
+@click.option(
+    '-t',
+    '--token',
+    default=os.environ.get('REANA_TOKEN', None),
+    help='The api key of an administrator.')
+@click.option(
+    '--json',
+    'output_format',
+    flag_value='json',
+    default=None,
+    help='Get output in JSON format.')
+@click.pass_context
+def get_users(ctx, id, email, user_token, token, output_format):
+    """Return user information."""
+    try:
+        response = _get_users(id, email, user_token, token)
+        headers = ['id', 'email', 'token']
+        data = []
+        for user in response:
+            data.append((str(user.id_), user.email, user.api_key))
+        if output_format:
+            tablib_data = tablib.Dataset()
+            tablib_data.headers = headers
+            for row in data:
+                tablib_data.append(row)
+
+            click.echo(tablib_data.export(output_format))
+        else:
+            click_table_printer(headers, [], data)
+
+    except Exception as e:
+        logging.debug(traceback.format_exc())
+        logging.debug(str(e))
+        click.echo(
+            click.style('User could not be retrieved: \n{}'
+                        .format(str(e)), fg='red'),
+            err=True)
+
+
+@users.command(
+    'create',
+    help='Create a new user.')
+@click.option(
+    '-e',
+    '--email',
+    help='The email of the user.')
+@click.option(
+    '-ut',
+    '--user-token',
+    help='The REANA_ACCESS_TOKEN of the user.')
+@click.option(
+    '-t',
+    '--token',
+    default=os.environ.get('REANA_TOKEN', None),
+    help='The api key of an administrator.')
+@click.pass_context
+def create_user(ctx, email, user_token, token):
```

Comment:
We could create non url safe tokens:
```console
$ flask users create -e diego@cern.ch -t <admin_token> -ut []?#/?][~~~.-#
$ flask users get -t <admin_token>
ID                                     EMAIL                     TOKEN
00000000-0000-0000-0000-000000000000   info@reana.io             <admin_token>
57aa5dd0-94d3-4640-96c7-5a479d4da95b   diego@cern.ch             []?#/?][~~~.-#
```
Shall we check if the token is url safe and if it is not warn the user?